getQuotedTweets.py

```diff
@@ -61,9 +58,12 @@ def get_referenced_tweets(referencedTweets):
 
 
 def main():
-    mentionsResponse = get_mentions()
+    bearerToken = os.getenv("tweetreceipttoken")
+    userId = os.getenv('tweetreceiptuserid')
+
+    mentionsResponse = get_mentions(userId)
     jmentionsResponse = mentionsResponse.json()
-    get_referenced_tweets(jmentionsResponse)
+    get_referenced_tweets(jmentionsResponse, bearerToken)
```

Comment:
@erickhammersmark Do we need to add bearer token as an input now to the function, in line 34?

Reply:
also in line 64, when calling get_mentions, I guess

Reply:
Sorry about that! Twitter was dragging their feet about turning on my API access and I didn't test until after I'd already done the 4th commit. Nice catch.